plugins/dynamic-plugins-info/src/index.ts

```diff
@@ -1 +1,3 @@
 export { dynamicPluginsInfoPlugin, DynamicPluginsInfo } from './plugin';
+export { default as PluginsInfoIcon } from '@mui/icons-material/PowerOutlined';
+export { default as AdminIcon } from '@mui/icons-material/GppMaybeOutlined';
```

Comment:
:+1:

I'm unsure, but is it required to export it also in plugins/dynamic-plugins-info/src/plugin.ts ??

Reply:
No, this works already. Same as how other plugin icons are exported.